src/java/org/apache/cassandra/db/marshal/AbstractType.java

```diff
@@ -118,8 +118,20 @@ protected AbstractType(ComparisonType comparisonType, boolean isMultiCell, List<
             throw new IllegalStateException();
         }
 
-        comparatorSet = new ValueComparators((l, r) -> compare(l, ByteArrayAccessor.instance, r, ByteArrayAccessor.instance),
-                                             (l, r) -> compare(l, ByteBufferAccessor.instance, r, ByteBufferAccessor.instance));
+        comparatorSet = new ValueComparators(new Comparator<>()
+        {
+            @Override
+            public int compare(byte[] l, byte[] r)
+            {
+                return AbstractType.this.compare(l, ByteArrayAccessor.instance, r, ByteArrayAccessor.instance);
+            }
+        }, new Comparator<>() {
```

Comment:
nit: Code style, braces on new line

Reply:
Thanks.

I really miss checkstyle, spotless...or any similar tool that does this check automatically. It is frustating for reviewers to highlight these issues.

Reply:
We have checkstyle, but indeed we do not have anything for the braces. I will take a look if I can add something later. 
Also, not frustrated on my end, I miss the braces all the time too... let's look at a checkstyle rule solution, that is a good idea so we do not have to think about that at all. :-)

Reply:
Bad news and good news - checkstyle was added in Apache Cassandra 4.1 and main is based on 4.0 so we do not have it now. BUT we will have it when we switch to newer version.